enterprise/backend/test/metabase_enterprise/serialization/v2/storage_test.clj

```diff
@@ -17,12 +17,25 @@
 
 (set! *warn-on-reflection* true)
 
+(defn- exclude-existing-files
```

Comment:
This title seems misleading, suggesting that we dynamically check for the pre-existing files.